ports/mimxrt/machine_pin.c

```diff
@@ -294,13 +245,13 @@ STATIC const mp_rom_map_elem_t machine_pin_locals_dict_table[] = {
     { MP_ROM_QSTR(MP_QSTR_PULL_HOLD), MP_ROM_INT(PIN_PULL_HOLD) },
 
     { MP_ROM_QSTR(MP_QSTR_DRIVER_OFF), MP_ROM_INT(PIN_DRIVE_OFF) },
-    { MP_ROM_QSTR(MP_QSTR_LOWEST_POWER), MP_ROM_INT(PIN_DRIVE_LOWEST_POWER) },
-    { MP_ROM_QSTR(MP_QSTR_LOW_POWER), MP_ROM_INT(PIN_DRIVE_LOW_POWER) },
-    { MP_ROM_QSTR(MP_QSTR_LOWER_POWER), MP_ROM_INT(PIN_DRIVE_LOWER_POWER) },
-    { MP_ROM_QSTR(MP_QSTR_MED_POWER), MP_ROM_INT(PIN_DRIVE_MED_POWER) },
-    { MP_ROM_QSTR(MP_QSTR_HIGHER_POWER), MP_ROM_INT(PIN_DRIVE_HIGHER_POWER) },
-    { MP_ROM_QSTR(MP_QSTR_HIGH_POWER), MP_ROM_INT(PIN_DRIVE_HIGH_POWER) },
-    { MP_ROM_QSTR(MP_QSTR_HIGHEST_POWER), MP_ROM_INT(PIN_DRIVE_HIGHEST_POWER) },
+    { MP_ROM_QSTR(MP_QSTR_DRIVE_POWER_0), MP_ROM_INT(PIN_DRIVE_POWER_0) }, // R0 (150 Ohm @3.3V / 260 Ohm @ 1.8V)
```

Comment:
For consistency, should `DRIVER_OFF` be renamed to `DRIVE_POWER_0`, and everything else here shifted up by 1?

Also, I always advocate for shorter names, and here I would consider `POWER0`, `POWER1`, etc.  In code that would look something like:
```python
p = Pin("GPIO_B_01", Pin.OUT, drive=Pin.POWER2)
```

Reply:
I really would like to keep the `DRIVE_OFF` option since this literally disables the output driver. With the rest I am fine and will fix it

Reply:
Fixed.